src/test/run-make-fulldeps/coverage-reports-base/expected_show_coverage.panic_unwind.txt

```diff
@@ -0,0 +1,50 @@
+    1|       |#![allow(unused_assignments)]
+    2|       |// expect-exit-status-101
+    3|       |
+    4|       |fn might_panic(should_panic: bool) {
+    5|      4|    if should_panic {
+    6|      1|        println!("panicking...");
+    7|      1|        panic!("panics");
+    8|      3|    } else {
+    9|      3|        println!("Don't Panic");
+   10|      3|    }
+   11|      3|}
+   12|       |
+   13|       |fn main() -> Result<(),u8> {
+   14|      1|    let mut countdown = 10;
+   15|     10|    while countdown > 0 {
+   16|     10|        if countdown == 1 {
+   17|      0|            might_panic(true);
+   18|     10|        } else if countdown < 5 {
+   19|      3|            might_panic(false);
+   20|     15|        }
```

Comment:
I just noticed this incorrect line count, `15`, here and at the same point in the `panic_unwind.rs` test.

This is wrong of course.

I suspect I have overlapping coverage regions with different counters, and they are adding up. (Best guess for now.) I'll look into it.

I think it may be related to how I implement the "hidden else", discussed in my previous reply to @tmandry in `abort.rs`. I may need to rethink what to do if the span is at the end of a line, and possible expand to the first character in the next line, if not at the end of the body.

(FYI, the expansion to the left is actually what I want for the end of the body. This is how I count function `Return`s. So I don't want to break that.)

Reply:
I fixed this when I fixed the `Goto` implicit else span, and remove the special case handling of `Goto` (which didn't work that well in some cases... like this one).